src/metrics/spatial.jl

```diff
@@ -122,3 +125,105 @@ function summarize(
 )::YAXArray where {D,T,N,A}
     return summarize(data[timesteps=timesteps], alongs_axis, metric)
 end
+
+"""
+    ensemble_loc_difference(outcome::YAXArray{T,3}, scens::DataFrame; agg_metric::Union{Function,AbstractFloat}=median, diff_target=:guided, conf::Float64=0.95, rand_seed=1234)::YAXArray where {T}
+
+Mean bootstrapped difference (counterfactual - target) between some outcome aggregated for
+each location.
+
+# Arguments
+- `outcome` : Metric outcome with dimensions (:timesteps, :locations, :scenarios).
+- `scens` : Scenarios DataFrame.
+- `agg_metric` : Metric used to aggregate scenarios when comparing between counterfactual and
+target. If it is an `AbstractFloat` between 0 and 1, it uses the `bs_metric`-th quantile.
+Defaults to `median`.
+- `diff_target` : Target group of scenarios to compare with. Valid options are `:guided` and
+`:unguided`. Defaults to `:guided`
+- `conf` : Percentile used for the confidence interval. Defaults to 0.95.
+- `rand_seed` : Seed used for random number generator. Defaults to 1234.
+
+# Example
+```
+# Load domain
+dom = ADRIA.load_domain(path_to_domain)
+
+# Create scenarios
+num_scens = 2^6
+scens = ADRIA.sample(dom, num_scens)
+
+# Run model
+rs = ADRIA.run_scenarios(dom, scens, "45")
+
+# Calculate difference to the counterfactual for given metric
+_relative_cover = metrics.relative_cover(rs)
+
+# Compute difference between guided and counterfactual using the 0.6-th quantile
+gd_res = metrics.ensemble_loc_difference(r_cover, scens; agg_metric=0.6)
+
+# Compute difference between unguided and counterfactual using the median
+ug_res = metrics.cf_difference_loc(r_cover, scens; diff_target=:unguided)
+
+# Plot maps of difference to the counterfactual
```

Comment:
```suggestion
# Plot maps of difference to the counterfactual at timestep 2
```

Is the above correct?

Reply:
Yes, this `2` is not timestep, `gd_res[1,:]` and `gd_res[3,:]` are the lower and upper bounds of the confidence interval and `gd_res[2,:]` is the metric used (median or whatever).

Reply:
Hmm, is `gd_res` a YAXArray? It would be clearer what the "2" is for the example if the axis name is used as the selector instead

Reply:
I'd ask that the comment be updated to clarify what the "2" is.